### DISCUSSION TITLE
Will it be possible to import key directly in AAXConverter

### DISCUSSION THREAD

**Discussion by Abish4i** [audiamus/AaxAudioConverter#121] [Ideas]
### I was thinking 
If AudibleActivation.sys workaround is still working will it be possible to incorporate some import feature for this file or modified version of same file in txt or other format.

**P.S** _I know about BookLibConnect it is great and all but I also want decrypt all those file already saved on my device_.

**Comment by audiamus** [comment]
As long as `AudibleActivation.sys` can be found at its designated location and it contains the activation code valid for your books, i.e the Audible Win 10 app must be activated for the matching account, AAX Audio Converter should not have problems to find the code.

You can also reveal the code in the basic settings and copy it to a safe place.

**Comment by Abish4i** [comment]
I have this file extracted from rooted Android device, though so far I'm unable to import keys to AAX Converter.

Will it work if I just make directory in Win10 and throw the file manually there.
**PS** Also I don't know what is default path ... xD

**Comment by audiamus** [comment]
I understand. 

> Will it work if I just make directory in Win10 and throw the file manually there.

Yes, you could do that. The default path is 
`C:\Users\<user>\AppData\Local\Packages\AudibleInc.AudibleforWindowsPhone_xns73kv1ymhp2\LocalState\AudibleActivation.sys`
 
You can also open the file in a binary editor (e.g. [Frhed](http://frhed.sourceforge.net/)). The first 4 bytes should be your key, but in reverse order.

**Comment by Abish4i** [comment]
I'll do that, thanks for helping
